tool/parseConfig.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"github.com/iotaledger/goshimmer/packages/parameter"
+)
+
+var (
```

Comment:
this parameters should be defined as `pflags` settings, as it is done within plugins and packages.